tests/pds/pds_common.go

```diff
@@ -335,6 +337,22 @@ func CleanUpBackUpTargets(projectID, prefix string) error {
 	return nil
 }
 
+func PerformRestore(restoreClient restoreBkp.RestoreClient, dsEntity restoreBkp.DSEntity, projectID string, deployment *pds.ModelsDeployment) []*pds.ModelsDeployment {
```

Comment:
why a new function to restore ? Existing function cannot be used ?

Reply:
This is a wrapper for restore functionality, this will reduce the number of lines in our test case.